### ISSUE TITLE
Issue With Installment

### ISSUE THREAD

**Issue by helloitsmeteeheehee** [theislab/diffxpy#169]
I keep getting the error:

---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
<ipython-input-6-bfb18af10d01> in <module>
----> 1 import diffxpy.api as de

ModuleNotFoundError: No module named 'diffxpy'

Whenever i try to import diffxpy or bash into my code. I think I installed it correctly (used pip) and cloned the folders into the same folder, so I don't know why this is happening.

**Comment by davidsebfischer** [comment]
can you install and import other packages @helloitsmeteeheehee ? i would suspect this is an evironment issue that could be diagnosed in this way.

**Comment by helloitsmeteeheehee** [comment]
yes, i can. it might be because i'm doing pip install -e into the wrong root directory.

should i use the general root directory for pip install -e? or should i be more specific and put it into a subroot (ie: Users)?

also, are diffxpy and bash supposed to be in the same folder?

also, is the jupyter notebook supposed to be opened inside of diffxpy or bash?

**Comment by davidsebfischer** [comment]
> should i use the general root directory for pip install -e? or should i be more specific and put it into a subroot (ie: Users)?

The directory does not matter, important is that you install for the same python installation.
 
> also, are diffxpy and bash supposed to be in the same folder?

No, bash is unrelated.

> also, is the jupyter notebook supposed to be opened inside of diffxpy or bash?

jupyter is installable as a python package, so as long as it s in the same python installation, you are fine!

I would encourage you to look up some tutorials on usage of conda and python environments, I think there are a couple of best practices you might be missing that would make your life a lot easier!
